src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -86,6 +87,30 @@
         return emails;
     }
     
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code student}
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmailForStudent(
+            FeedbackSessionAttributes session, StudentAttributes student) {
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        EmailWrapper email = generateFeedbackSessionEmailBaseForStudentSubmissionConfirmation(course, session,
+                student);
+        return email;
+    }
+
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code instructor}.
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmailForInstructor(
+            FeedbackSessionAttributes session, InstructorAttributes instructor) {
+        
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        EmailWrapper email = generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(course, session,
+                instructor);
```

Comment:
You could just move the contents of `generateFeedbackSessionEmailBaseFor*SubmissionConfirmation` to the relevant `generateFeedbackSubmissionConfirmationEmailFor*` since they are only used there.